test/DurableTask.SqlServer.Tests/Integration/DatabaseManagement.cs

```diff
@@ -319,19 +326,16 @@ public async Task CanCreateIfNotExists(bool isDatabaseMissing)
         [Theory]
         [InlineData(true)]
         [InlineData(false)]
-        public void SchemaCreationIsSerializedAndIdempotent(bool isDatabaseMissing)
+        public async Task SchemaCreationIsSerializedAndIdempotent(bool isDatabaseMissing)
         {
             using TestDatabase testDb = this.CreateTestDb(!isDatabaseMissing);
             IOrchestrationService service = this.CreateServiceWithTestDb(testDb);
 
             // Simulate 4 workers starting up concurrently and trying to initialize
             // the same database schema. It should just work with predictable output.
-            Parallel.For(0, 4, i =>
-            {
-                service.CreateIfNotExistsAsync().GetAwaiter().GetResult();
-            });
+            await Enumerable.Range(0, 4).ParallelForEachAsync(4, i => service.CreateIfNotExistsAsync());
```

Comment:
The xunit change added some new analyzers that raise compiler errors if we do things like block threads with calls like `Task.GetAwaiter().GetResult()`. That's why I had to change this line of code. It should be mostly equivalent to the previous `Parallel.For` call.